db/models.py

```diff
@@ -50,3 +59,60 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
```

Comment:
I think we should follow this datetime format `2022-03-15 15:59:50` please check if you comply everywhere